app/pages/settings/email.jsx

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
+import Translate from 'react-translate-component';
+import counterpart from 'counterpart';
 import apiClient from 'panoptes-client/lib/api-client';
 import talkClient from 'panoptes-client/lib/talk-client';
 import AutoSave from '../../components/auto-save';
 import handleInputChange from '../../lib/handle-input-change';
 
+counterpart.registerTranslations('en', {
+  zooPrefs: {
+    general: 'Get general Zooniverse email updates',
+    classify: 'Get email updates from the Projects you classify on',
+    beta: 'Get beta project email updates and become a beta tester'
+  },
+  talkPrefs: {
+    title: 'Send me an email',
+    frequency: {
+      immediate: 'Immediately',
+      day: 'Daily',
+      week: 'Weekly',
+      never: 'Never'
+    }
```

Comment:
@eatyourgreens the talk preferences options are missing here because they're inside the lookup function, so I haven't figured out how to add them yet.

Reply:
Language strings should be added to `app/locales/en.js`, not embedded in the component.

Reply:
For the Talk preferences, you need to remove the `nameOfPreference` method and replace it with references to counterpart keys instead eg. `<Translate content="settings.talk.participating_discussions" />`.